js/app.js

```diff
@@ -30,7 +30,7 @@
 		$stateProvider
 			.state('home', {
 				url: '/',
-				templateUrl: 'pages/home/view.html',
+				templateUrl: 'pages/home/home.view.html',
```

Comment:
hmmm... is that a real home view... or is just an entry state? For me, a home is a state after crossing the login/register or whatever is the main screen of your app

Reply:
Actually, that's not the home, but the main page. I don't know if it will be an empty state, or will be the new home, or whatever. But... What is clear is that we shall follow [the naming guide](https://github.com/Videona/platform-frontend/wiki/Component-structure) we spent time making hahaha.